community/kernel/src/main/java/org/neo4j/kernel/impl/store/AbstractDynamicStore.java

```diff
@@ -254,11 +255,21 @@ public int getNumberOfReservedLowIds()
     @Override
     protected void readAndVerifyBlockSize() throws IOException
```

Comment:
This is not important for this PR but it might be in the long plan of cleaning up the store hierarchy: this method is only used in `https://github.com/neo4j/neo4j/blob/2.3/community/kernel/src/main/java/org/neo4j/kernel/impl/store/CommonAbstractStore.java#L210` so it should be made private and moved there. This can be of course be part of another PR.